### PR TITLE
msid in tracks

### PR DESCRIPTION
**Pull request by FelonEkonom**
Adjusting code to related changes from membraneframework/membrane_webrtc_plugin#8

### CODE REVIEW

File: webrtc/videoroom/assets/src/membraneWebRTC.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const phoenix_channel_push_result = async (push: Push): Promise<any> => {
 
 interface Participant {
   displayName: string;
-  mids: string[];
+  ids: string[];
```
Comment (Qizot) [inline webrtc/videoroom/assets/src/membraneWebRTC.ts:16]: Probably should be trackIds. `ids` does not tell much what it is.

File: webrtc/videoroom/assets/src/membraneWebRTC.ts [hunk 2]
```diff
@@ -226,16 +226,15 @@ export class MembraneWebRTC {
   private onTrack = () => {
     return (event: RTCTrackEvent) => {
       const [stream] = event.streams;
-      const mid = event.transceiver.mid!;
-      const isScreenSharing = mid.includes("SCREEN") || false;
+      const id = event.track.id;
+      const isScreenSharing = id.includes("SCREEN") || false;
```
Comment (Qizot) [inline webrtc/videoroom/assets/src/membraneWebRTC.ts:230]: Have you checked that screensharing works? I tried to wrap my head around this pr and one in webrtc plugin and got lost easily...

**Review by Qizot — CHANGES_REQUESTED** [review]

File: webrtc/videoroom/lib/videoroom/pipeline.ex [hunk 3]
```diff
@@ -169,6 +158,49 @@ defmodule VideoRoom.Pipeline do
...
+
+    state = %{state | display_engine: %DisplayEngine{state.display_engine | endpoints: endpoints}}
+
+    mid_to_msid =
+      Map.get(state, :mid_to_msid, %{})
+      |> Map.merge(mid_to_msid)
+
+    state =
+      state
+      |> Map.put(:mid_to_msid, mid_to_msid)
```
Comment (mickel8) [inline webrtc/videoroom/lib/videoroom/pipeline.ex:179]: Why do we store this in state? I think we don't need these lines

**Review by mickel8 — CHANGES_REQUESTED** [review]

**Comment by mickel8** [comment]
Outdated

**Pull request closed without merging**